python/ray/_private/runtime_env/pip.py

```diff
@@ -51,6 +51,11 @@ def get_virtualenv_python(cls, target_dir: str) -> str:
         virtualenv_path = cls.get_virtualenv_path(target_dir)
         return os.path.join(virtualenv_path, "bin/python")
 
+    @classmethod
+    def get_virtualenv_activate(cls, target_dir: str) -> str:
+        virtualenv_path = cls.get_virtualenv_path(target_dir)
+        return os.path.join(virtualenv_path, "bin/activate")
```

Comment:
This should be `Script/activate` on windows

Reply:
I can make it part of #22449 if this gets merged before that PR

Reply:
@mattip Sounds good! Thank you very much! I don't have windows environment. 😓